bindings/Clutter/ChangeLog.md

```diff
@@ -0,0 +1,4 @@
+1.26.4
```

Comment:
This is not very important, but generally for the version of the bindings we use the version of the gir file + revision. So in this case I would suggest something like `1.0.1` (meaning gir version `1.0`, first version of the bindings for this version). The same comment applies to the other bindings below.